django_session_jwt/middleware/session.py

```diff
@@ -172,6 +173,20 @@ def process_request(self, request):
             request.session['jwt'] = fields
 
     def process_response(self, request, response):
+        if isinstance(request.user, AnonymousUser):
```

Comment:
Here you can do `request.user.is_authenticated`, django docs specify that is the way to detect `AnonymousUser` since the class is an implementation detail.